slash/src/base_conf.cc

```diff
@@ -126,13 +126,14 @@ bool BaseConf::GetConfInt(const std::string &name, int* value) const {
   return false;
 }
 
-bool BaseConf::GetConfInt64(const std::string &name, long long int* value) const {
+bool BaseConf::GetConfInt64(const std::string &name, int64_t* value) const {
   for (int i = 0; i < rep_->item.size(); i++) {
     if (rep_->item[i].type == Rep::kComment) {
       continue;
     }
     if (name == rep_->item[i].name) {
-      (*value) = atoi(rep_->item[i].value.c_str());
+      char *end;
```

Comment:
Dangerous here, since 'end' stays uninitialized, but 'strtoll' will try to stores the address of the first invalid character in it when it is not NULL.

So just leave the second parameter of ‘strtoll’ NULL, if we doesn't care that.